src/Hotel.java

```diff
@@ -32,24 +37,20 @@ int hotelNumberOfRooms(){
         return availableRooms;
     }
 
-    /*we can't know the perfect available room for the guests
-    sometimes all "perfect" rooms are booked but the quests don't mind sleeping together in two personas bed
-    or the opposite
-     */
-    Room findPerfectFitRoomOrReturnNull(Date from, Date to, int numberOfCouples, int numberOfPeopleSleepingAlone){
-        int numberOfPeople = numberOfCouples*2 + numberOfPeopleSleepingAlone;
+    Room findPerfectFitRoomOrReturnNull(Date from, Date to, int personas)
+    {
         int thereIsPerfectRoom = 0;
```

Comment:
In Java we use booleans instead of int. It is hard getting use to starting from C and C++, I know. :)